arangod/Pregel/AlgoRegistry.cpp

```diff
@@ -159,6 +162,9 @@ template<typename V, typename E, typename M>
   } else if (algorithm == "wcc") {
     return createWorker(vocbase, new algos::WCC(server, userParams), body,
                         feature);
+  } else if (algorithm == "readwrite") {
+    return createWorker(vocbase, new algos::ReadWrite(server, userParams), body,
+                        feature);
```

Comment:
Wonder whether one should:

```suggestion
#if defined(ARANGODB_ENABLE_MAINTAINER_MODE)
  } else if (algorithm == "readwrite") {
    return createWorker(vocbase, new algos::ReadWrite(server, userParams), body,
                        feature);
#endif
```